tests/foreman/cli/test_organization.py

```diff
@@ -1693,3 +1694,29 @@ def test_positive_info_by_name(self):
         org = make_org()
         result = Org.info({'name': org['name']})
         self.assertEqual(org['id'], result['id'])
+
+    @tier1
+    def test_positive_multibyte_latin1_org_names(self):
+        """Hammer Multibyte and Latin-1 Org names break list pagination
+
+        :id: 4fa0afe7-6d0a-4c3e-a0fc-4ecb95c50fc9
+
+        :BZ: 1418412
+
+        :expectedresults: Multibyte and latin1 names need to be
+            displayed with consistent spacing
+        """
+        org_names = valid_org_names_list()
+        for org in org_names:
+            make_org({'name': org})
+        org_list = [line for line in Org.list(output_format='table') if line]
+        self.assertGreaterEqual(len(org_list), len(org_names))
+        for org_str in org_list:
+            width = 0
+            for char in org_str:
+                width = sum(
+                    1 if unicodedata.east_asian_width(char)
+                    in ["Na", "N", "A", "H"]
+                    else 2 for char in org_str
```

Comment:
Not exactly :) The comprehension inside `sum()` contains a loop already which will navigate through each character in line. This way you don't need extra loop `for char in org_str:` outside, it basically breaks all logic which currently sounds like `for each char in line calculate sum of all characters inside that char ` :)